### PR TITLE
mail/fetchmail: New plugin

### PR DESCRIPTION
**Pull request by mimugmail**
This is a new plugin, helping e.g. for migrations from older hosted imap/pop3 to local exchange.

### CODE REVIEW

File: mail/fetchmail/src/etc/inc/plugins.inc.d/fetchmail.inc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+function fetchmail_services()
+{
+    global $config;
+
+    $services = array();
+
+    if (isset($config['OPNsense']['fetchmail']['general']['enabled']) && $config['OPNsense']['fetchmail']['general']['enabled'] == 1) {
+        $services[] = array(
+            'description' => gettext('Postfix'),
```
Comment (fichtner) [inline mail/fetchmail/src/etc/inc/plugins.inc.d/fetchmail.inc:37]:
```suggestion
            'description' => gettext('Fetchmail'),
```

File: mail/fetchmail/src/opnsense/mvc/app/views/OPNsense/Fetchmail/mailbox.volt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        $("#saveAct_progress").addClass("fa fa-spinner fa-pulse");
+            ajaxCall(url="/api/fetchmail/service/reconfigure", sendData={}, callback=function(data,status) {
+                updateServiceControlUI('fetchmail');
+                $("#saveAct_progress").removeClass("fa fa-spinner fa-pulse");
+            });
+        });
+    });
+
+});
+</script>
```
Comment (fichtner) [inline mail/fetchmail/src/opnsense/mvc/app/views/OPNsense/Fetchmail/mailbox.volt:91]:
```suggestion
</script>
```

**Review by fichtner — APPROVED** [review]
Better late than never, looks good!

**Comment by fichtner** [comment]
Merged, thanks ❤️

**Comment by mimugmail** [comment]
Impressive merge pace 👍

**Comment by mimugmail** [comment]
Ah, needs chmod as usual, also fetchmailrc needs 600. Thx

**Comment by fichtner** [comment]
> Impressive merge pace 👍

Phone is pretty sloppy but free time is key. You guys are doing all the work really. 😊

Will do the cleanups, change logs and version bumps on Monday.

**Comment by mimugmail** [comment]
@fichtner sorry to write here, just a small one. Would you mind setting permissions of fetchmailrc?

`File /usr/local/etc/fetchmailrc must have no more than -rwx------ (0700) permissions.`

**Comment by fichtner** [comment]
Git can only do 755.. I guess it needs to be added to setup.sh

**Pull request merged**